File: Python Word Examples/ReplaceTextApiDemo/replaceWithRegex.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ReplaceTextApi(configuration)

file_name = 'ReplaceText.docx'
regex = '\\#\\w+\\b'
new_value = 'E-iceblue'
extra = {'folder': "input", 'dest_file_path': 'output/ReplaceWithRegex.docx'}
result =api.replace_with_regex(file_name, regex, new_value, **extra)

File: JavaScript Word Examples/ReplaceTextApiDemo/replaceWithRegex.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ReplaceTextApi(apiClient);

    var filename = "ReplaceWithRegex.docx";
    var regex = "\\#\\w+\\b";
    var newValue = 'E-iceblue';
    var destFilePath = 'output/ReplaceWithRegex_output.docx';
    var opt = { folder: 'input' };
    instance.replaceWithRegex(filename, regex, newValue, destFilePath, opt, function (error, data, response) {
        if (error) throw error;
    });
})();